docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -20,50 +20,67 @@ date: "2022-10-11"
+These `fake` components interact with Viam like real hardware but do not physically exist.
 
-Most Viam [components](../../../components/) have a _fake_ model that you can use for testing.
-These fake components interact with Viam like real hardware but do not actually exist.
+### What is a sub-part?
 
-In this tutorial, you will set up, control, and program a mock robotic arm and a mock sub-part with a motor using fake components.
+Usually, when building a {{< glossary_tooltip term_id="robot" text="robot" >}}, you pick out a [single-board computer](/components/board/) like the [Jetson Nano](/components/board/jetson/) or [Raspberry Pi](/components/board/pi/).
+You follow the instructions in the **Setup** tab to install `viam-server` on your [board](/components/board/), and you start operating your robot with that computer, adding the [components](/components/) and [services](/services/) you want to use to that `viam-server` instance.
 
-## Requirements
+By utilizing {{< glossary_tooltip term_id="part" text="parts" >}}, you can expand upon this, chaining multiple computers together to build your robot with Viam:
+
+- Each individual computer-controlled unit of a robot is called a “{{< glossary_tooltip term_id="part" text="part" >}}” in Viam.
+- Typically, simple robots have just one part, but you can have as many parts as your project requires.
+- Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
+- You can access any sub-part either directly, or through any part above it in the tree.
+- Each part runs a single `viam-server` instance.
+
+### What is a `fake` component?
+
+Most Viam [components](../../../components/) have a `fake` model that you can use for testing.
+These `fake` components interact with Viam like real hardware but do not actually exist.
+
+## Set up a mock robot
 
 You'll need the following hardware and software for this tutorial:
 
-- A laptop or desktop running Linux or macOS
+- A computer running Linux or macOS
 - [Go](https://go.dev/doc/install) or [Python 3.9+](https://www.python.org/downloads/)
-- A code editor of your choice
 
-## Set up a mock robot
+If you don't already have an account, create a free one.
+Sign up on [the Viam app](https://app.viam.com).
+Create a new robot in the organization and location of your choice.
+Go to this robot's **Setup** tab.
 
-### Make an account on the Viam app
+### Install and start `viam-server` on your computer
 
-The first thing you need to do is set up your account on the Viam app.
-Go to [app.viam.com](https://app.viam.com) and sign up for a free account.
+Before you proceed with configuring and controlling your robot, install `viam-server`.
+Follow the steps outlined for your computer's architecture on the **Setup** tab of the [Viam app](https://app.viam.com) to install `viam-server` on your computer as a system service.
+Find more information on running `viam-server` in the [installation guide](/installation/manage/).
```

Comment:
```suggestion
Follow the steps outlined for your computer's architecture on the **Setup** tab of the [Viam app](https://app.viam.com) to [install `viam-server`](/installation/manage/) on your computer as a system service.
```
I think the extra sentence might make people go to that page - when they probably don't need to - I think slightly less likely like this

Reply:
Done, sounds good to me